app/code/Magento/Deploy/Strategy/QuickDeploy.php

```diff
@@ -131,4 +136,28 @@ private function getInheritanceLevel(Package $package)
         }
         return $level;
     }
+
+    /**
+     * Verify if can deploy theme
+     *
+     * @param string $theme
+     * @param array $includedThemesMap
+     * @param array $excludedEntitiesMap
+     * @return bool
+     */
+    private function canDeployTheme(string $theme, array $includedThemesMap, array $excludedEntitiesMap): bool
+    {
+        $includesAllThemes = \array_key_exists('all', $includedThemesMap);
```

Comment:
Using here `\array_key_exists` instead of `isset` because of optimization https://github.com/php/php-src/pull/3360 that was done in php 7.4.